utils/libnetwork/Makefile

```diff
@@ -30,7 +30,7 @@ define Package/libnetwork
   CATEGORY:=Utilities
   TITLE:=networking for containers
   URL:=https://github.com/docker/libnetwork
-  DEPENDS:=$(GO_ARCH_DEPENDS)
+  DEPENDS:=$(GO_ARCH_DEPENDS) @!(mips||mipsel||mips64||mips64el)
```

Comment:
This bothers me since the buildbot cannot run. I have no idea if https://github.com/moby/moby/pull/37490 can be backported. Should be fine for now.

Reply:
As stated in https://github.com/openwrt/packages/pull/9547#issuecomment-520114436 I would first focus attention to containerd after which backporting might be unnecessary.